Packs/CortexXDR/Layouts/layoutscontainer-Cotrex_XDR_Device_Control_Violations.json

```diff
@@ -0,0 +1,313 @@
+{
+    "detailsV2": {
+        "tabs": [
+            {
+                "id": "summary",
+                "name": "Legacy Summary",
+                "type": "summary"
+            },
+            {
+                "hidden": false,
+                "id": "zmjvpqxkcs",
+                "name": "XDR Device Control Violations",
```

Comment:
@EliorKedar Also please move the layout here
![image](https://user-images.githubusercontent.com/51712181/103876080-e1209180-50db-11eb-9195-8e3e765093f3.png)